worlds/_bizhawk/context.py

```diff
@@ -235,8 +235,12 @@ async def _run_game(rom: str):
 
 
 async def _patch_and_run_game(patch_file: str):
-    metadata, output_file = Patch.create_rom_file(patch_file)
-    Utils.async_start(_run_game(output_file))
+    try:
+        metadata, output_file = Patch.create_rom_file(patch_file)
+        Utils.async_start(_run_game(output_file))
+    except (FileNotFoundError, ValueError) as exc:
```

Comment:
`Patch.create_rom_file` could go through any implementation of `APPatch` that any game could implement with any code.

So, would this be better?
```python
except BaseException as exc:
```

Reply:
Wanted to give a more specific error message in the case where the mistake was likely that the user used the wrong ROM or forgot it. Forgot that `logger.exception` exists though, which should give enough info anyway, and will still catch and display other problems.

Seems like `except Exception` is more appropriate here than `except BaseException`, but I'm no expert.

Reply:
My understanding of the difference between `BaseException` and `Exception` is that `BaseException` includes things that aren't really meant to be handled.

So the fact that you're not handling anything here and just reraising the exception made me think `BaseException` was more appropriate.

Reply:
Well, those things that "aren't really meant to be handled" still have no real reason to be handled here. The `BaseExceptions` I see in the python docs don't really look like things worth intercepting or logging to the user. My purpose in re-raising them was so that the actual exception and trace weren't lost to the log file just because I nabbed them for a user-facing message. Now that I'm just directly logging them, I'm not sure there would be a point in re-raising them anyway.